.readthedocs.yml

```diff
@@ -17,5 +17,5 @@ build:
 python:
   install:
     - method: pip
-      path: .
+      path: .[docs]
```

Comment:
Are you trying to install the docs dependencies here? It's confusing to put this [docs] in path.

You can either add them to doc/requirements_docs.txt or use extra_requirements: -docs like here: https://docs.readthedocs.io/en/stable/config-file/v2.html#packages